src/communication/stream_transfer_sender.cpp

```diff
@@ -147,6 +150,11 @@ namespace cubstream
 	cubcomm::er_log_debug_buffer ("transfer_sender::read_action", ptr, byte_count);
 
 	m_last_sent_position += byte_count;
+
+	if (m_p_stream_ack)
```

Comment:
Is this a temporary solution  for ACK ?
I remember that the agreed solution was that slave server notify the master of the latest received position, while this is just the last position pushed to send buffer.

Reply:
Is a temporary solution. Will be moved after recv, when communication channel will be ready.